src/plugins/home/public/application/components/home_app.js

```diff
@@ -92,6 +93,12 @@ export function HomeApp({ directories, solutions }) {
           <Route exact path="/feature_directory">
             <FeatureDirectory addBasePath={addBasePath} directories={directories} />
           </Route>
+          <Route exact path="/homepage">
+            <PlaygroundHomepage />
+          </Route>
+          <Route exact path="/olly-homepage">
```

Comment:
nit: `olly`

because i believe it will be `/home/olly-homepage` which seems redundant